packages/account/globals.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+    const content: any;
+    export default content;
+}
```

Comment:
What is this file used for?

Reply:
in javascript webpack it is allowing the svg files to be call in the component but in TypeScript it is not accepting the svg files in the content so we have to declare this to let TypeScript know that we need to import svg files into our content